src/store/reducers/counterReducers.js

```diff
@@ -1,11 +1,11 @@
 import { INCREMENT, DECREMENT, RESET} from '../actions/counterActions';
```

Comment:
move it ti the separate file like "src/store/actionTypes.js"

Reply:
https://github.com/mtalstykh-exadel/team4_frontend/blob/27f79d12b2ded8e5b31bd2b44a4a2d0ffd8adfa8/src/store/actions/actionTypes.js#L1-L9